README.md

```diff
@@ -2403,6 +2406,52 @@ and 'absent'.
 Name of the ospf router. Valid value is a string.
 
 --
+### Type: cisco_ospf_area
+
+Manages an area for an OSPF router.
+
+| Platform | OS Minimum Version | Module Minimum Version |
+|----------|:------------------:|:----------------------:|
+| N9k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N3k      | 7.0(3)I3(1)        | 1.4.0                  |
+| N5k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N6k      | 7.3(0)N1(1)        | 1.4.0                  |
+| N7k      | 7.3(0)D1(1)        | 1.4.0                  |
+| N8k      | 7.3(0)F1(1)        | 1.4.0                  |
+

```

Comment:
I would like to see some example usage documentation also. This is a gap in our docs. I did this for ACE: https://github.com/saichint/cisco-network-puppet-module/blob/a67dd1c7efc2bf014bedee5102092e8ca9736c52/README.md#example-usage

So, maybe just some basic examples like this:

```
#### Example Usage

cisco_ospf_area { 'my_ospf_instance  default  0':
  ensure          => 'present',
  range           => [['10.3.0.0/16', 'not_advertise', '23'],
                            ['10.3.3.0/24', '450']
                           ],
}

cisco_ospf_area { 'my_ospf_instance  my_vrf  1.1.1.1':
  ensure          => 'present',
  authentication  => 'md5',
  default_cost    => 1000,
  filter_list_in  => 'fin',
  filter_list_out => 'fout',
  stub             => true,
}

#### Parameters


| Example Parameter Usage |
|:--|:--
|`cisco_ospf_area { '<ospf_process_id> <vrf> <area_id>':`
|`cisco_ospf_area { '1 my_vrf 0':`
|`cisco_ospf_area { 'my_ospf default 10.1.1.1':`

```